c/extras/py_ftrl.cc

```diff
@@ -254,156 +384,173 @@ void Ftrl::set_model(robj model) {
                        << "` and `" << dt_model_in->columns[1]->stype() << "`";
   }
 
-  fm->set_model(dt_model_in);
+  ft->set_model(dt_model_in);
 }
 
 
 /*
 *  All other getters and setters.
 */
-oobj Ftrl::get_a() const {
-  return py::ofloat(fm->get_a());
+oobj PyFtrl::get_colnames_hashes() const {
+  size_t n_features = ft->get_n_features();
+  py::otuple py_colnames_hashes(n_features);
+  std::vector<uint64_t> colnames_hashes = ft->get_colnames_hashes();
+  for (size_t i = 0; i < n_features; ++i) {
+    py_colnames_hashes.set(i, py::oint(static_cast<size_t>(colnames_hashes[i])));
+  }
+
+  return std::move(py_colnames_hashes);
 }
 
 
-oobj Ftrl::get_b() const {
-  return py::ofloat(fm->get_b());
+oobj PyFtrl::get_params() const {
+  strpair tuple_info;
+  tuple_info.first = "Params";
+  tuple_info.second = "FTRL model parameters";
+
+  py::onamedtupletype nttype(tuple_info, Type::params_info);
```

Comment:
This should really be a static member of `PyFtrl` class, initialized on first access.

Reply:
Yes, sure. Still have some work to be done on the tuple/namedtuple/docs here.

Reply:
Fixed